docs/src/postmortem/2021-06-06.md

```diff
@@ -0,0 +1,93 @@
+Post Mortem 2021-06-06
+======================
```

Comment:
Everywhere else in the docs we use the `#` style for setting up headings.  Please change this document to follow suit.